### DISCUSSION TITLE
Добавить  share public  link доступ для ресурсов

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1144] [Ideas]
По аналогии с google docs, желательно добавить метод для ресурса ->withShareLink(), который бы отображал ссылку, по которой любой пользователь без авторизации может посмотреть сущность в режиме просмотра, при этом оболочка и лейаут не грузиться, видна только запись и все.   (пример: турагентство присылает пользователю в месседжер ссылку на проверку правильности вбитых в moonshine ресурс паспортных данных)

**Comment by lee-to** [comment]
не сочтите за наглость но даже комментировать не хочу
